src/Hacknet/HashUpgrade.ts

```diff
@@ -43,7 +44,7 @@ export class HashUpgrade {
   hasTargetCompany = false;
 
   /** Name of upgrade */
-  name = "";
+  name = "" as HashUpgradeEnum;
```

Comment:
We used to set this to enjoy string because it was a quick way to set the type. Now that empty string isn't a valid value, this might be better:
```suggestion
  name: HashUpgradeEnum;
```